### DISCUSSION TITLE
WAS keeps downgrading transformers and pillow

### DISCUSSION THREAD

**Discussion by Davikar** [WASasquatch/was-node-suite-comfyui#221] [General]
Is there any specific reason that WAS node suite requires transformers==4.26.1 and Pillow==9.5?
These packages are downgraded every time I update the WAS node suite, so every time I have to go back and update them again.
Seems to run fine with the newer versions anyway.

**Comment by WASasquatch** [comment]
BLIP requires older transformers. Pillow probably doesn't need to be pinned anymore since colabs has updated their python installs.
